### PR TITLE
Update integration tests and gas estimates logic

### PR DESCRIPTION
**Pull request by ra-phael**
- Removes deprecated eth gas station in favor of BlockNative
- Restricts to the USDC/WETH @ 0.04% for gas estimate pricing
- Makes sure all integration tests pass

### CODE REVIEW

File: src/routers/alpha-router/config.ts [hunk 1]
```diff
@@ -69,5 +69,5 @@ export const DEFAULT_ROUTING_CONFIG_BY_CHAIN = (
       };
   }
 };
-export const ETH_GAS_STATION_API_URL =
-  'https://ethgasstation.info/api/ethgasAPI.json';
+
+export const BLOCKNATIVE_APIKEY = 'bba51317-1a55-4e83-9ea5-7a6dc3014795';
```
Comment (henrik-web3) [inline src/routers/alpha-router/config.ts:73]: Env var?

File: src/providers/blocknative-gas-price-provider.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  baseFee: number;
+}
+
+export class BlockNativeGasPriceProvider extends IGasPriceProvider {
+  private apiKey: string;
+  private url: string;
+  constructor(apiKey: string) {
+    super();
+    this.apiKey = apiKey;
+    this.url = 'https://api.blocknative.com/gasprices/blockprices';
```
Comment (henrik-web3) [inline src/providers/blocknative-gas-price-provider.ts:45]: Env var?

File: test/integ/routers/alpha-router/alpha-router.integration.test.ts [hunk 3]
```diff
@@ -944,16 +881,17 @@ describe('alpha router integration', () => {
...
-            SwapType.SWAP_ROUTER_02,
-            quote,
-            tokenIn,
-            tokenOut,
-            methodParameters,
-            tradeType,
-            100,
-            100
-          );
+          // We can't actually execute a swap against mainnet without an EAT
```
Comment (0xpApaSmURf) [inline test/integ/routers/alpha-router/alpha-router.integration.test.ts:884]: Probably worth adding this comment everywhere the `validateExecuteSwap` is commented out for clarity

**Review by 0xpApaSmURf — APPROVED** [review]

File: src/providers/blocknative-gas-price-provider.ts (hunk 2 above)
Comment (ra-phael) [reply src/providers/blocknative-gas-price-provider.ts:45]: We discussed it with papasmurf and decided against as the extra burden is not worth it

**Pull request merged**